src/Nethermind/Nethermind.Facade/BlockchainBridge.cs

```diff
@@ -248,12 +256,11 @@ private TransactionResult CallAndRestore(
             BlockHeader blockHeader,
             Transaction transaction,
             bool treatBlockHeaderAsParentBlock,
-            ITxTracer tracer)
+            ITxTracer tracer,
+            IOverridableTxProcessingScope scope)
```

Comment:
Minor: here and `TryCallAndRestore`. I think `scope` should be earlier in parameters, either 1st or 3rd. Would make it more readable and have most important parameters first.